### DISCUSSION TITLE
NocoDB  French tutorial [Apr 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1907] [Discord Threads]
> 🙋‍♂️ **Richnou**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Thank you so much Richard for making this tutorial.  
#  
> 👨‍🚀 **wingkwong**  
#### merci beaucoup  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/965991111044788235)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
